server/settings.py

```diff
@@ -196,3 +196,6 @@
 # Create hashed+gzipped versions of assets during collectstatic,
 # which will then be served by WhiteNoise with a suitable max-age.
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+
+# Add a MIME type for .wasm files (which is not included in WhiteNoise's defaults)
+WHITENOISE_MIMETYPES = {'.wasm': 'application/wasm'}
```

Comment:
Is this something we should (also) address upstream?

Reply:
That's not a bad idea.